Source/miss2_0x11C.cpp

```diff
@@ -71,20 +89,38 @@ void miss2_0x11C::SCRCMD_CAR_DECSET_503BC0(s32* a1, s32 a2)
 {
 }
 
-STUB_FUNC(0x503f80)
+MATCH_FUNC(0x503f80)
 void miss2_0x11C::SCRCMD_PARKED_CAR_DECSET_503F80(s32 a1)
 {
+	s32 v1;
+    miss2_0x11C::SCRCMD_CAR_DECSET_503BC0((s32 *)a1, a1);
+    (*(Car_BC **)(a1 + 8))->sub_443EB0(9);
```

Comment:
This and the other places should really use the correct class instead of hardcoded offsets, so I'd recommend trying to figure them out next time or picking functions that have already sane-ish looking code.